### PR TITLE
Fix test cases

### PR DESCRIPTION
**Pull request by cryptoAtwill**
Fix broken test cases.

Closes ENG-816

### CODE REVIEW

File: fendermint/testing/contract-test/tests/staking/machine.rs [hunk 1]
```diff
@@ -137,10 +137,17 @@ impl StateMachine for StakingMachine {
...
-
+            eprintln!(
+                "\n> JOINING SUBNET: addr={_addr} deposit={}",
+                v.power.collateral
+            );
+
+            let v = Validator {
+                public_key: v.public_key.clone(),
+                power: Collateral(v.power.collateral.clone()),
+            };
```
Comment (aakoshh) [inline fendermint/testing/contract-test/tests/staking/machine.rs:148]: Why is this needed? It seems to do nothing.

**Comment by aakoshh** [comment]
So, once again I disagree where this is going, but if the PR introduces (breaking!) changes to the genesis structure and the CLI, I think it should come with a way more elaborate title and description than "Fix test cases".

**Comment by cryptoAtwill** [comment]
Temporary closing this as we will refactor this process.
